### DISCUSSION TITLE
How to add border to CTkLabel?

### DISCUSSION THREAD

**Discussion by duruburak** [TomSchimansky/CustomTkinter#1450] [Q&A]
Is it possible to add a border to CTkLabel, like we can do to tkinter.Label?

```python
import tkinter as tk

root = tk.Tk()

label = tk.Label(root, text="Hello, world!", bd=2, relief="solid")
label.pack()

root.mainloop()
```

**Comment by bfedie518** [comment]
Doesn't seem like you can.  The most you can do is change the `fg_color` and `corner_radius`.

**Comment by bfedie518** [comment]
I added relief and borderwidth/bd in pull request #1467

**Comment by Akascape** [comment]
@duruburak 
For now, use a button with `hover=False` and `fg_color="transparent"` to make it look like a lable.
